cnames_active.js

```diff
@@ -1590,7 +1590,8 @@ var cnames_active = {
   "pong-zh": "xiaozhu2021.github.io/javascript-pong",
   "popcorn-api": "nirewen.github.io/popcorn-api",
   "popper": "fezvrasta.github.io/popper.js",
-  "portafolioacadeller": "acadeller.github.io/portafolioacadeller", 
+  "portafolioacadeller": "acadeller.github.io/portafolioacadeller",
```

Comment:
```suggestion
  "portafolioacadeller": "acadeller.github.io/portafolioacadeller", 
```
Please revert the change to this line, to allow for a clean blame.